packages/web-app-files/src/views/SharedWithMe.vue

```diff
@@ -256,3 +457,28 @@ export default {
   }
 }
 </script>
+
+<style>
+.centered {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+.shares-bar {
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+}
+
+#pending-highlight {
+  background-color: aliceblue;
+}
+.show-hide-pending {
+  text-align: center;
+  padding-bottom: 12px;
```

Comment:
Please don't use pixel values for padding / margin. You could either use our margin and padding classes from the design system on the element where you added the `show-hide-pending` class. I.e. remove the padding from the class here and add the respective padding class on the button where you used it. Documentation is available here: https://owncloud.design/#/Utilities/Padding The class that is interesting here for you is `oc-pb-s`, which stands for `**p**adding **b**ottom **s**mall`.

Another way would be to use the css variable for padding sizes.

This is important so that your theming config regarding spacing will take effect here as well.

Reply:
You can see the default spacing pixel sizes here: https://owncloud.design/#/Design%20Tokens